### ISSUE TITLE
custom loss function behaving different from build-in loss

### ISSUE THREAD

**Issue by lukasfolle** [keras-team/keras#11223]
Hi,

I just noticed, that when defining a custom loss-function:
`def custom_loss(y_true, y_pred): return keras.losses.binary_crossentropy(y_true, y_pred)`
The optimization is much slower than using `'binary_crossentropy'` directly as loss in the compile function.
Info: I am training a modified U-Net version with Tensorflow as backend.

I would like to know if this is just me or someone else experienced something similar before.

Lukas

**Comment by yanboliang** [comment]
@lufol Do you mean the compiling time is slow or the running time? Can you provide a simple code snippet to reproduce it ? Thanks.

**Comment by brot4** [comment]
Hi, I had the same problem. After restarting the Jupyter Notebook, it worked fine.
